src/evaluate-all-cells.js

```diff
@@ -0,0 +1,19 @@
+import actions from './actions'
+
+export default function evaluateAllCells(cells, store) {
+  cells.forEach((cell) => {
```

Comment:
I'm going to make a bold claim here. The FIRST set of cells that should be run is actually the CSS Cells, if possible, then the markdown cells. Otherwise we get a flickering between the default styles and the ones applied via CSSCells. I believe we'd do it right here in this function. It's also possible that we can very easily just do all the markdown + css cells together and that'd take care of the issue!

Reply:
Yep, that seems reasonable, and easily changed